mapillary/utils/verify.py

```diff
@@ -195,3 +196,39 @@ def points_traffic_signs_check(kwargs: dict) -> dict:
             "existed_at": kwargs.get("existed_at", None),
             "existed_before": kwargs.get("existed_before", None),
         }
+
+
+def valid_id(id: int, image=True):
+    """Checks if a given id is valid as it is assumed. For example, is a given id expectedly an
+    image_id or not? Is the id expectedly a map_feature_id or not?
+
+    :param id: The ID passed
+    :type id: int
+
+    :param image: Is the passed id an image_id?
+    :type image: bool
+
+    '''
+    :raises InvalidOptionError: Raised when invalid arguments are passed
+    '''
+
+    :return: None
+    :rtype: None
+    """
+
+    # IF image == False, and error_check == True, this becomes True
+    # IF image == True, and error_check == False, this becomes True
+    if image ^ EntityAdapter().is_image_id(id=id, fields=[]):
```

Comment:
What does the `^` do in this context?

Reply:
`^` is the XOR operator in `python`. So it only becomes true when both the operands have different values of boolean, like below,

![image](https://user-images.githubusercontent.com/41635766/127892515-d99e5c29-ee3a-4fc3-aba6-bfc3d6cb65bd.png)